src/Microsoft.ML.Data/EntryPoints/InputBase.cs

```diff
@@ -37,10 +44,13 @@ public enum CachingOptions
     public abstract class LearnerInputBase
     {
         /// <summary>
-        /// The data to be used for training.
+        /// The data to be used for training. Used only in entry-points, since in the API the expected mechanism is
+        /// that the user iwll use the <see cref="IEstimator{TTransformer}.Fit(IDataView)"/> or some other train
```

Comment:
typeo- `iwll`

Reply:
> Did you mean to check this `core_ep-list.tsv.bak` file in?

No. @Ivanidzo4ka had the same comment, I think if you refresh the view you're using you should note that it is removed.

Reply:
> typeo- `iwll`

Whoops. Since internal only I will fix in next phase, if that's OK.